code/modules/mob/living/blood.dm

```diff
@@ -66,17 +66,34 @@
 
 
 		// Blood regens using food, more food = more blood.
-		if(blood_volume < BLOOD_VOLUME_NORMAL)
-			switch(nutrition)
-				if(NUTRITION_OVERFED to INFINITY)
-					adjust_nutrition(-10)
-					blood_volume += 1 // regenerate blood quickly.
-				if(NUTRITION_HUNGRY to NUTRITION_OVERFED)
-					adjust_nutrition(-5)
-					blood_volume += 0.5 // regenerate blood slowly.
-				if(0 to NUTRITION_HUNGRY)
-					adjust_nutrition(-1)
-					blood_volume += 0.1 // Regenerate blood VERY slowly.
+		switch(blood_volume)
+			if(BLOOD_VOLUME_SAFE to BLOOD_VOLUME_NORMAL) //Passively regens blood very slowly from 90% to 100% without a tradeoff.
+				blood_volume += 0.1
+			if(BLOOD_VOLUME_OKAY to BLOOD_VOLUME_SAFE) //Regens blood from 50% ish to 90% using nutrition.
+				switch(nutrition)
+					if(NUTRITION_OVERFED to INFINITY)
+						adjust_nutrition(-10)
+						blood_volume += 1 // regenerate blood quickly.
+					if(NUTRITION_HUNGRY to NUTRITION_OVERFED)
+						adjust_nutrition(-5)
+						blood_volume += 0.5 // regenerate blood slowly.
+					if(0 to NUTRITION_HUNGRY)
+						adjust_nutrition(-1)
+						blood_volume += 0.1 // Regenerate blood VERY slowly.
+			if(BLOOD_VOLUME_SURVIVE to BLOOD_VOLUME_OKAY)
```

Comment:
The toxloss from this is very high, and low blood already has damage applications innately. Beyond that, I'm not convinced that dividing nutrition effects between low and very low blood is helpful either for comprehension or gameplay. The safe to normal buffer zone and base regen is good and should address the main issue.